### DISCUSSION TITLE
How to automatically insert a pagebreak if a table is going to break across a page in PDF output?

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [quarto-dev/quarto-cli#10750] [Q&A]
### Description

Is there a way to automatically insert a pagebreak whenever a table is going to be split across two pages when outputting to PDF (Latex)? I presently render it and then go about inserting pagebreaks manually, rerendering because other tables get shifted and then inserting more pagebreaks until there are no tables split across pages. Was wondering if there's a more elegant solution.

**Comment by mcanouil** [comment]
I don't think there is, especially with the float system of LaTeX.

**Reply by lokesh-krishna** [reply]
The float system is what's confusing me.My understanding of floats was that their purpose was to prevent tables and figures from being broken over a page. But that's what's not happening. Do I need to enable floats as a PDF option or something?

**Reply by mcanouil** [reply]
That's LaTeX. There is no fine grained control
over how floats are moved or not.
Note that this is not a Quarto question as it is a known (for decades) LaTeX issue.
You need to play around with positioning and inserting page break.

https://quarto.org/docs/authoring/markdown-basics.html#page-breaks
